src/internal/stream/priority_queue.go

```diff
@@ -47,7 +47,7 @@ func (pq *PriorityQueue) Iterate(cb func([]Stream, ...string) error) error {
 func (pq *PriorityQueue) isHigherPriority(i, j int) bool {
 	si := pq.queue[i]
 	sj := pq.queue[j]
-	return si.Key() < sj.Key() || (si.Key() == sj.Key() && si.Priority() < sj.Priority())
+	return si.Key() < sj.Key() || (si.Key() == sj.Key() && si.Priority() > sj.Priority())
```

Comment:
Why was this made?

Reply:
We had two implementations of this abstraction, one for the fileset merge and one for the datum merge (skipping). I merged them (pun intended) into one abstraction and had to choose a priority ordering. I went with the way the fileset merge determined priority (higher value is higher priority), so I changed the stream package implementation which was only used by the datum merge originally (I also had to change the datum merge logic to account for this).

Reply:
Okay so any comments which talk about merging left to right, where leftmost is the lowest index and rightmost is the last index still make sense?

Reply:
Yeah, I'm going to include this in the file parts removal PR. I'm tweaking this again with the goal of just having a consistent ordering for the items going in and coming out of the merge. The inversion we do during the merge is confusing and error prone.